Assets/Scripts/Minimap.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Cosmobot
+{
+    public class Minimap : MonoBehaviour
+    {
+        public Transform player;
+        public GameObject minimap;
```

Comment:
I've always had dilemma with `public` vs `private` in Unity. Certainly `player` should be `private` (add `[SerializeField]` attribute to display field in inspector) because it is an "internal" field for use only by this script and no one should have the ability to see or change it.

As for the `minimap`, we can allow other scripts change its value, but idk if we want that behavior here... For now, set to `private`.

Reply:
Changed to private